manifests/state-ipoib-cni/0050-ipoib-cni-ds.yml

```diff
@@ -74,9 +74,12 @@ spec:
       volumes:
         - name: cnibin
           hostPath:
+            # First we try to set the user-set value, then fallback to defaults for Openshift / K8s
```

Comment:
lets do this logic in the state and not the template.
i think it should be applied generically for all relevant (CNI related) states, then we  just template the value.

1. if  provided - take the value
2. else
2.1  if openshift -> /var/lib/cni/bin
2.2 else -> /opt/cni/bin